CONTRIBUTING.md

```
@@ -0,0 +1,168 @@
+
+
+## Contribute Code
+
+We like code commits a lot! They're super handy, and they keep the project going and doing the work it needs to do to be useful to others.
+To contribute code:
+
+* [Set up the project](#project-setup).
+* Make any necessary changes to the source code.
+* Write tests that verify that your contribution works as expected.
+* Write clear, concise commit message(s).
+* Go to https://github.com/canopas/group-track-android/pulls and open a new pull request with your changes.
```

Markdownlint (0.37.0)

Bare URL used

(MD034, no-bare-urls)